source/agora/node/Ledger.d

```diff
@@ -888,7 +888,7 @@ public class Ledger
                           block.header.height, K, idx);
                 return "Block: Couldn't find pre-image for validator";
             }
-            Point R = CR + Scalar(preimage_info.hash).toPoint();
+            Point R = Scalar(preimage_info.hash) * CR;
```

Comment:
```suggestion
            Point R = CR * Scalar(preimage_info.hash);
```
Better to change the order of the operands as other statements.

Reply:
I applied your change as it then matches the slide.